package.json

```diff
@@ -60,7 +60,7 @@
     "@hmcts/nodejs-healthcheck": "^1.6.0",
     "@hmcts/properties-volume": "^0.0.9",
     "@hmcts/rpx-xui-common-lib": "0.1.38",
-    "@hmcts/rpx-xui-node-lib": "^2.23.4",
+    "@hmcts/rpx-xui-node-lib": "^2.24.0",
```

Comment:
just a small update, but could you bump again?

Reply:
@paddybasi  this is done now. Thanks.